src/components/TestDetailsModal.tsx

```diff
@@ -152,6 +152,7 @@ const TestDetailsModal: React.FunctionComponent<{
   );
 
   useHotkeys("d", () => setIsDiffShown((isDiffShown) => !isDiffShown));
+  useHotkeys("ESC", () => handleClose());
```

Comment:
could you make separate PR for this?

Reply:
https://github.com/Visual-Regression-Tracker/Visual-Regression-Tracker/issues/174